.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -10,9 +10,6 @@ body:
   - type: markdown
     attributes:
         value: Please fill in the fields below, deleting the sections that don't apply to your issue.
```

Comment:
Not sure how this would render, but this information would help a lot.
```suggestion
        value: Before continuing, please check the issue tracker for existing issues and https://scipy.github.io/devdocs/index.html for the development documentation. Your issue could be already fixed on the development version. If you have an installation problem, make sure that you are following the latest installation recommendations: https://scipy.github.io/devdocs/dev/contributor/contributor_toc.html#development-environment 
```

Reply:
This is how the render would be: 

![image](https://user-images.githubusercontent.com/47351025/131549959-ccf1b3ed-653e-4c21-9646-674943fb026a.png)